recipes/gnupg/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "gnupg" %}
+{% set version = "2.2.4" %}
+{% set sha256 = "401a3e64780fdfa6d7670de0880aa5c9d589b3db7a7098979d7606cec546f2ec" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://www.gnupg.org/ftp/gcrypt/gnupg/gnupg-{{ version }}.tar.bz2
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script:
+    - export CPPFLAGS="-DIN_EXCL_UNLINK=0x04000000 $CPPFLAGS"  # [linux]
+    - export LDFLAGS="-lrt $LDFLAGS"                           # [linux]
+    - ./configure --prefix="$PREFIX"
+    - make
+    - make install
+  skip: true  # [win]
+  run_exports:
+    - {{ pin_subpackage('gnupg', max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - bzip2
+    - sqlite
+    - libgcrypt
+    - libgpg-error
```

Comment:
Are these two unneeded in run?

Reply:
both of those have been modified to run_exports